test/zdtm.py

```diff
@@ -2697,6 +2701,9 @@ def get_cli_args():
     rp.add_argument("--noauto-dedup",
                     help="Manual deduplicate images on iterations",
                     action='store_true')
+    rp.add_argument("--skip-zero-pages",
```

Comment:
@simonis Would you be able to also enable testing with `--skip-zero-pages` in [run-ci-tests.sh](https://github.com/checkpoint-restore/criu/blob/criu-dev/scripts/ci/run-ci-tests.sh)?

Reply:
@simonis  It was great meeting you at FOSDEM and your talk was very good!

We can use something like the following patch to run the existing ZDTM tests with `--skip-zero-pages`: https://github.com/rst0git/criu/commit/45a8ca12b5e708e9dcb72e208621ee2405e27a87

Reply:
@avagin Thank you for the review! Would it be sufficient to run the following tests?
```diff
diff --git a/scripts/ci/run-ci-tests.sh b/scripts/ci/run-ci-tests.sh
index ef7e869e0..8f5e25d03 100755
--- a/scripts/ci/run-ci-tests.sh
+++ b/scripts/ci/run-ci-tests.sh
@@ -268,6 +268,9 @@ make -C test/others/rpc/ run
 ./test/zdtm.py run -t zdtm/transition/maps007 --pre 2 --page-server --dedup
 ./test/zdtm.py run -t zdtm/transition/maps007 --pre 2 --pre-dump-mode read
 
+# Run tests with --skip-zero-pages
+./test/zdtm.py run --skip-zero-pages -T '.*maps0.*'
+
 ./test/zdtm.py run -t zdtm/transition/pid_reuse --pre 2 # start time based pid reuse detection
 ./test/zdtm.py run -t zdtm/transition/pidfd_store_sk --rpc --pre 2 # pidfd based pid reuse detection

```